Update instruction for short lived container (#4283)
DataDog/documentation: content/en/agent/docker/_index.md

**Note**: The `docker.containers.running`, `.stopped`, `.running.total` and `.stopped.total` metrics are not affected by these settings and always count all containers. This does not affect your per-container billing.

#### Short Lived containers

> By default, the agent looks for new containers every 5 seconds. Therefore any container that has a shorter duration life might not be picked up by the agent autodiscovery.

Reviewer: ```suggestion
By default the Agent looks every 5 seconds for new containers, any container with a shorter duration life won't have any data collected by the Agent.
```

> One option to solve that situation is to override this value thanks to the `ad_config_poll_interval` parameter which correspond to the `DD_AD_CONFIG_POLL_INTERVAL` environment variable.

Reviewer: ```suggestion
You can override this autodiscovery interval with a shorter one by setting `ad_config_poll_interval` parameter which correspond to the `DD_AD_CONFIG_POLL_INTERVAL` environment variable.
```

The expected value is a integer in seconds.

> A workaround could also be to configure the container to wait for at least the poll interval before starting.

Reviewer: ```suggestion
A better workaround would be to configure your containers to wait for at least the poll interval before starting.
```

> A workaround could also be to configure the container to wait for at least the poll interval before starting.

Reviewer: @NBParis @hkaj what's the benefit of the sleep if we can just override the ad interval?

Author: I believe that we should. 

The idea behind is decreasing the interval increase the log on the Kubelet API and therefore on your cluster.
Also no matter which interval you put, a container can fail sooner which is why using the sleep is a good workaround until we manage to get all the containers that started since the last poll.

> This can be achieved by adding a sleep period that is as long as the poll interval:`sh -c 'sleep 5; exec yourcommand --etc`

Reviewer: ```suggestion
This can be achieved by adding a sleep period that is as long as the poll interval:`sh -c 'sleep 5; exec <COMMAND> --etc`
```
